src/idsk/components/intro-block/intro-block.scss

```diff
@@ -24,52 +24,77 @@
         }
 
         .govuk-body{
-                margin-bottom: govuk-spacing(4);
+            margin-bottom: govuk-spacing(4);
             @include mq($from: desktop) {
                 margin-bottom: govuk-spacing(8);
             }
-        
         }
 
-        .app-pane-grey {
+        div.app-pane-grey {
+          @include mq($from: desktop) {
             background-color: govuk-colour("light-grey");
+          }
+          @include mq($until: tablet) {
+            padding: 0 govuk-spacing(3);
```

Comment:
Nedavat odsadenie na zaklade triedy ktora meni farbu backgroundu.

Reply:
No ale tu farba nie je jediné čo sa mení.. Ja to beriem skôr ako typ. Pretože modrý background má aj dots v liste a šedý nemá.. Čiže nemá ani odsadenie